deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -25,6 +30,35 @@ Parameters:
     Type: String
     Description: The path to look for the application config from
 
+  AcmCertificateArn:
```

Comment:
You can probably just import this from the InfrastructureStackName

Reply:
I will probably wait to see what happens with Tom's DNS work before I go too far into this section, for now, it is fine with me to use the cloud front url.